ui/widgets/tooltip.js

```diff
@@ -143,25 +145,29 @@ $.widget( "ui.tooltip", {
 		} );
 
 		// Remove title attributes to prevent native tooltips
-		this.element.find( this.options.items ).addBack().each( function() {
-			var element = $( this );
-			if ( element.is( "[title]" ) ) {
-				element
-					.data( "ui-tooltip-title", element.attr( "title" ) )
-					.removeAttr( "title" );
-			}
-		} );
+		this.disabledTitles = this.disabledTitles.add(
```

Comment:
That would result in `this.disabledTitles` being an empty set after `.disable().disable()` using default settings because `this.options.items` looks for elements with titles, but the titles are removed after the first call.